vulnerabilities/models.py

```diff
@@ -39,75 +39,87 @@ class Vulnerability(models.Model):
     A software vulnerability with minimal information. Identifiers other than CVE ID are stored as
     VulnerabilityReference.
     """
-    cve_id = models.CharField(
-        max_length=50, help_text='CVE ID', unique=True, null=True)
-    summary = models.TextField(
-        help_text='Summary of the vulnerability', blank=True)
-    cvss = models.FloatField(max_length=100, help_text='CVSS Score', null=True)
+
+    cve_id = models.CharField(max_length=50, help_text="CVE ID", unique=True, null=True)
+    summary = models.TextField(help_text="Summary of the vulnerability", blank=True)
+    cvss = models.FloatField(max_length=100, help_text="CVSS Score", null=True)
 
     @property
     def vulnerable_to(self):
-        return self.packagerelatedvulnerability_set.filter(is_vulnerable=True)
+        return self.package_set.filter(
+            packagerelatedvulnerability__is_vulnerable=True,
+            packagerelatedvulnerability__vulnerability_id=self.id,
```

Comment:
I don't think this lookup is necessary since the related manager is starting on `self`.